libcloud/ecp.cfg.orig

```diff
@@ -34,6 +34,8 @@ libcloud:
         search {{ lab_domain }}
         nameserver {{ nameserver_addr }}
         EOF
+        sed -i 's/NETCONFIG_DNS_STATIC_SEARCHLIST=""/NETCONFIG_DNS_STATIC_SEARCHLIST="{{ lab_domain }}"/' /etc/sysconfig/network/config
```

Comment:
To avoid duplication I wonder if you could do here like:
```
sed -Ei 's/(NETCONFIG_DNS_STATIC_SEARCHLIST)=""/\1="{{ lab_domain }}"/' /etc/sysconfig/network/config
```